src/fixtures/nodoRPTResponses.ts

```diff
@@ -306,3 +306,64 @@ export const activateV2PaymenNoticeResponseAllCCPlight = (): MockResponse => [
     </soapenv:Body>
     </soapenv:Envelope>`
 ];
+
+export const activateV2PaymenNoticeResponseWithConventionMetadata = (): MockResponse => [
+  200,
+  `<soapenv:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:common="http://pagopa-api.pagopa.gov.it/xsd/common-types/v1.0.0/" xmlns:nfp="http://pagopa-api.pagopa.gov.it/node/nodeForPsp.xsd">
+    <soapenv:Body>
+        <nfp:activatePaymentNoticeV2Response>
+            <outcome>OK</outcome>
+            <totalAmount>100.00</totalAmount>
+            <paymentDescription>Quota Albo Ordine Giornalisti - ${Math.floor(
+              Math.random() * 99999999999
+            )}</paymentDescription>
+            <fiscalCodePA>77777777775</fiscalCodePA>
```

Comment:
```suggestion
            <fiscalCodePA>77777777774</fiscalCodePA>
```
should this be valued with the request fiscal code?

Reply:
correct!